recipe/meta.yaml

```diff
@@ -61,9 +61,13 @@ test:
     - conda_build
     # new/updated submodules (can be dropped after 1-2 releases)
     - conda_build.index
+  requires:
+    - setuptools
```

Comment:
This feels weird to me. Is it because pip check complains that setuptools isn't present?

Reply:
setuptools was in the other version of the recipe/feedstock. Maybe for `- test_bdist_conda_setup.py`

Reply:
this is copied over from the AnacondaRecipes conda-build-feedstock where it was originally added in https://github.com/AnacondaRecipes/conda-build-feedstock/pull/26, not sure why